CMake/ctkMacroSetupQt.cmake: A duplicate ELSE command was found inside an IF block

Hi there,
when using CTK with on macOS sierra 10.12.5 with latest cmake --version `cmake version 3.8.20170518-gbc7d64` , I get the error message:

```
CMake Error at CMake/ctkMacroSetupQt.cmake:76 (else):
  A duplicate ELSE command was found inside an IF block.
Call Stack (most recent call first):
  CMakeLists.txt:410 (ctkMacroSetupQt)
```

Indeed it seems there is a duplicate `else()` in the `ctkMacroSetupQt.cmake`.
Suggesting to remove the lines (that seem duplicated from an earlier merge years ago):
https://github.com/commontk/CTK/blob/master/CMake/ctkMacroSetupQt.cmake#L76-L77

- CTK revision 7129bb0b65d52c8ad6d5940be7aed3075b675edc (latest in master as of writing this).

--- a/CMake/ctkMacroSetupQt.cmake
+++ b/CMake/ctkMacroSetupQt.cmake
@@ -72,9 +72,6 @@ macro(ctkMacroSetupQt)
     else()
       message(FATAL_ERROR "error: Qt4 was not found on your system. You probably need to set the QT_QMAKE_EXECUTABLE variable")
     endif()
-
-  else()
-    message(FATAL_ERROR "error: Qt4 was not found on your system. You probably need to set the QT_QMAKE_EXECUTABLE variable")
   endif()
 
   mark_as_superbuild(CTK_QT_VERSION)

